starter-code/javascript/main.js

```diff
@@ -51,10 +59,25 @@ function setResetBtn() {
 
 // Start/Stop Button
 btnLeft.addEventListener('click', function () {
-
+    if ( btnLeft.classList.contains("start") ){
+        chronometer.startClick(printTime)
```

Comment:
startClick no necesita que le paséis una función! startClick inicial el crono en la app y printTime hace que se renderice gráficamente